internal/safety/safety_test.go

```diff
@@ -32,7 +32,7 @@ var goleakIgnoreOptions = []goleak.Option{
 }
 
 func TestMain(m *testing.M) {
-	log.Init()
+	log.Init(log.WithLoggerType("nop"))
```

Comment:
should we use string via const value to avoid corrections due to naming changes? 🤔 
```suggestion
	log.Init(log.WithLoggerType(logger.NOP.String()))
```